configure.ac

```diff
@@ -293,6 +293,15 @@ if test "$enable_man" = "yes"; then
 fi
 AM_CONDITIONAL(ENABLE_REGENERATE_MAN, test "x$enable_man" != "xno")
 
+if test "$enable_subids" != "no"; then
+	AC_CHECK_HEADERS([sys/prctl.h], [], [
+		if test "x$enable_subids" = "xyes"; then
+			AC_MSG_ERROR([Cannot enable support the subordinate IDs on systems which do not have sys/prctl.h])
+		fi
+		enable_subids="no"
```

Comment:
Please do not override value passed form command line parameter.

It is much better to inform user about optional feature for `support the subordinate IDs`
and let them decide how to solve it. Either to install missing header or to disable feature.
e.g.
```
checking for sys/prctl.h... no
configure: error: Cannot enable support the subordinate IDs on systems which do not have sys/prctl.h
                             The feature is optional and can be disabled by parameter --disable-subordinate-ids

```

Reply:
> Please do not override value passed form command line parameter.
This is exactly like below: if enable_subids != no mean it's either an explicit yes or no command line. If it's an explicit yes we error out, so the only case where enable_subids=no is done is in the no command line.

> or to disable feature
> can be disabled by parameter --disable-subordinate-ids

It's rather "not require the feature". This case is when the users explicitly asked for --enable-subordinate-ids, so we can only tell them, "well, sorry, no, we cannot enable it as you requested" The feature being optional is also fully known since the user explicitly asked for it.